token-swap/program/src/processor.rs

```diff
@@ -784,7 +784,7 @@ impl Processor {
                 to_u128(swap_token_b.amount)?,
                 pool_mint_supply,
                 trade_direction,
-                RoundDirection::Floor,
+                LiquidityProviderOperation::Withdrawal,
```

Comment:
Should be `Deposit` -- finally something makes sense!

Reply:
done